src/record/export/utils/operateFilename.ts

```diff
@@ -0,0 +1,4 @@
+// https://learn.microsoft.com/en-us/windows/win32/fileio/naming-a-file
```

Comment:
IMO: You can simplify the file name like `utils/file.ts`

Reply:
Fixed with 31671f3348b0138d9c98d158321636ac2f2f7588